geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/collections/IndexibleTreeSet.java

```diff
@@ -47,4 +49,9 @@ public E get(int index) {
   public int indexOf(E element) {
     return headSet(element).size();
   }
+
+  @Override
+  public ArrayList<E> getIndexRange(int min, int max) {
```

Comment:
The return type should just be a `List` here.

Reply:
Obviated by changing the return type to Iterator<E>.